affiliatewp-sign-up-bonus.php

```diff
@@ -48,6 +48,24 @@ class AffiliateWP_SB_Requirements_Check extends AffiliateWP_Requirements_Check {
 	 */
 	protected $slug = 'affiliatewp-sign-up-bonus';
 
+	/**
+	 * Add-on requirements.
+	 *
+	 * @since 1.0.0
```

Comment:
Fix the since version to 1.2.
```suggestion
	 * @since 1.2
```